### ISSUE TITLE
Markdown tag with ** or __ doesn't work

### ISSUE THREAD

**Issue by Silvan87** [geany/geany#1497]
It would be very useful if markdown tag with **bold* or __italic__ changed color text. It doesn't work while other text editors do it.

**Comment by elextr** [comment]
Seems whoever made the mappings for markdown mapped lots of syntax elements to the same style in the [code](https://github.com/geany/geany/blob/84253714771f48dbc7fab02f7de43f253734dee2/src/highlightingmappings.h#L995) instead of mapping each to a different style and left it up to the configuration files to do the mapping.

Pull requests fixing it are welcome.

**Comment by codebrainz** [comment]
On 2017-05-17 06:34 AM, Silvan87 wrote:
> It would be very useful if markdown tag with **bold* or __italic__ changed color text. It doesn't work while other text editors do it.
>

You can change it in your colour scheme .conf file. "strong" is mapped 
to "string_3" and "emphasis" is mapped to "string_4".

Regards,
Matthew Brush

**Comment by Silvan87** [comment]
So, markdown tags ** and __ do work, but if you have to change colorscheme, this is uncomfortable. Maybe it's better other default choice for markdown colors.

**Comment by codebrainz** [comment]
I agree, if someone were to create a pull request against Geany and Geany-Themes to improve it, it would be welcome.

**Comment by b4n** [comment]
The thing is that we don't yet have style that have the semantic of most markup languages, so there isn't something very sensible to map them to.  Ideally I guess we'd have additional styles specific for that.  I think we had a discussion started about that, but I can't seem to find it just now.
But yeah I guess it makes sense to have those styles, the only problem I know is that it'll break every theme that don't define those new names styles.

**Comment by elextr** [comment]
@b4n, it the new names are defined in `filetypes.common` won't that be used if the name isn't defined in the colour scheme?  Sure it may be ugly, especially for a dark colour scheme, but that will make it obvious that the colour scheme needs updating :)

**Comment by codebrainz** [comment]
There was https://github.com/geany/geany-themes/pull/2, but I think it was also discussed somewhere else.

**Comment by b4n** [comment]
Since #1837 strong and emphasis styles have respectively the **bold** or *italic* property set on them, regardless of the colorscheme.  Is that sufficient to close this issue, or some extra styles are indeed required?

**Comment by ralf3u** [comment]
@codebrainz Excuse me, I have a question to your comment https://github.com/geany/geany/issues/1497#issuecomment-302266150: How do I make it, that for example `**ha ha**` is not shown bold in the editor? Thank you in advance for an answer.
